### PR TITLE
OSDOCS-8517-Installer p2 bug text for 4.15 GA RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15 

Issue: [OSDOCS-8517](https://issues.redhat.com/browse/OSDOCS-8517)

Link to docs preview: [Start at Bug fixes and scroll until you get to the new installer bugs.](https://71996--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes)

QE review:
Not needed

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1546,6 +1546,29 @@ With this fix, the installation program now displays an error message that speci
...
 
+* Previously, Agent-based installations on {oci-first} did not show a console displaying installation progress to users, making it more difficult to track installation progress. With this update, Agent-based installations on {oci} now display installation progress on the console. (link:https://issues.redhat.com/browse/OCPBUGS-19092[*OCPBUGS-19092*])
+
+* Previously, if static networking was defined in the `install-config.yaml` or `agent-config.yaml` files for the Agent-based Installer, and an interface name was over 15 characters long, the network manager did not allow the interface to come up. With this update, interface names longer than 15 characters are truncated and the installation can proceed. (link:https://issues.redhat.com/browse/OCPBUGS-18552[*OCPBUGS-18552*])
+
+* Previously, if the user didn't specify the `rendezevousIP` field in the `agent-config.yaml` file and hosts were defined in the same file with static network configuration, then the first host was designated as a rendezvous node regardless of its role. This caused the installation to fail.
```
Comment (tmalove) [inline release_notes/ocp-4-15-release-notes.adoc:1553]:
```suggestion
* Previously, if the user didn't specify the `rendezevousIP` field in the `agent-config.yaml` file and hosts were defined in the same file with static network configuration, then the first host was designated as a rendezvous node regardless of its role. This caused the installation to fail.
```
Contractions are allowed, but I suggest expanding if you have time.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1546,6 +1546,29 @@ With this fix, the installation program now displays an error message that speci
...
+With this update, the Agent-based Installer prioritizes the rendezvous node search by first looking amongst the hosts with a `master` role and a static IP defined. If none is found, then a potential candidate is searched for through the hosts that do not have a role defined. Hosts with a static network configuration that are explicitly configured with a `worker` role are ignored. (link:https://issues.redhat.com/browse/OCPBUGS-5471[*OCPBUGS-5471*])
```
Comment (tmalove) [inline release_notes/ocp-4-15-release-notes.adoc:1554]:
```suggestion
With this update, the Agent-based Installer prioritizes the rendezvous node search by first looking among the hosts with a `master` role and a static IP defined. If none is found, then a potential candidate is searched for through the hosts that do not have a role defined. Hosts with a static network configuration that are explicitly configured with a `worker` role are ignored. (link:https://issues.redhat.com/browse/OCPBUGS-5471[*OCPBUGS-5471*])
```

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**